server/tests/controllers/metrics_controller/test_calc_metrics_deployments.py

```diff
@@ -395,3 +406,67 @@ async def test_deployment_metrics_empty_granularities(self, sample_deployments):
         )
         res = await self.post_json(json=body, assert_status=400)
         assert "granularities" in res["detail"]
+
+
+class TestChangeFailureMetrics(BaseCalcMetricsDeploymentsTest):
+    async def test_base(self, mdb_rw: Database, rdb: Database, sdb: Database) -> None:
+        body = self._body(
+            date_from="2005-01-01",
+            date_to="2005-03-01",
+            for_=[{}],
+            metrics=[
+                DeploymentMetricID.DEP_COUNT,
+                DeploymentMetricID.DEP_CHANGE_FAILURE_COUNT,
+                DeploymentMetricID.DEP_CHANGE_FAILURE_RATIO,
+            ],
+        )
+        DeplFact = DeploymentNotificationFactory
+        DeplCompFact = DeployedComponentFactory
+        DeplLabelFact = DeployedLabelFactory
+        await models_insert(
+            rdb,
+            DeplFact(name="d1", finished_at=dt(2005, 1, 10), started_at=dt(2005, 1, 1)),
```

Comment:
Can we please use either "Dep" or "Deployment" prefix. Not blocking, just for the future note. Two reasons:
- "l" sucks to read anyway
- We use "dep" in the main code, it is more widespread. This must be the same anti-logic as shortening "repository" to "repo" instead of "rep".